src/Stripe.net/Entities/PaymentMethods/PaymentMethodCardWallet.cs

```diff
@@ -0,0 +1,34 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class PaymentMethodCardWallet : StripeEntity
+    {
+        [JsonProperty("amex_express_checkout")]
+        public PaymentMethodCardWalletAmexExpressCheckout AmexExpressCheckout { get; set; }
+
+        [JsonProperty("apple_pay")]
+        public PaymentMethodCardWalletApplePay ApplePay { get; set; }
+
+        [JsonProperty("dynamic_last4")]
+        public string DynamicLast4 { get; set; }
+
+        [JsonProperty("google_pay")]
+        public PaymentMethodCardWalletGooglePay GooglePay { get; set; }
+
+        [JsonProperty("masterpass")]
+        public PaymentMethodCardWalletMasterpass Masterpass { get; set; }
+
+        [JsonProperty("microsoft")]
```

Comment:
Should this be `microsoft_pay`?

Reply:
In theory it should. It wasn't when I did the PR and flagged it internally :p 
Also it's possible it will not be in the final PR!